SPECS/nghttp2/nghttp2.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        nghttp2 is an implementation of HTTP/2 and its header compression algorithm, HPACK.
 Name:           nghttp2
-Version:        1.57.0
```

Comment:
How confident are you in this version bump?  Are there any compatibility concerns?

Reply:
@mfrw kind ping?

Reply:
I am slightly iffy about this version bumper given what we experienced with git & git-lfs lately.

Reply:
Superseded by:
- https://github.com/microsoft/azurelinux/pull/10656